design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -13,6 +13,8 @@ type Mutation {
 }
 
 input RequestReturnInput {
+    order_id: ID!
```

Comment:
```suggestion
    order_uid: ID!
```

Suggestion from @nrkapoor, pending discussion.

Reply:
Discussed with @nrkapoor and agree to replace `id` with `uid` everywhere in schema for returns.